scripts/cnode-helper-scripts/env

```diff
@@ -844,6 +844,27 @@ getProtocolParams() {
   [[ ${PROT_MAJOR} -eq 0 ]] && return 1 || return 0
 }
 
+getProtocolParamsHist() {
```

Comment:
1. subarg is a special cncli variable, it looks like for how its called that $1 should be used here instead.
2. cli doesn't allow for historical parameter lookup, so if KOIOS API is not available function should return a failcode. Currently it will just continue and process PROT_PARAMS variable anyway. 
3. dont print error, just return with error code and let the calling code print error if needed.

Reply:
fixed and testing

Reply:
getProtocolParamsHist in env file has been updated as requested.